frontend/src/components/atoms/IconText.tsx

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 
 interface IconTextProps {
-  icon: React.ReactElement;
+  icon: any;
```

Comment:
I think the reason I changed this was because the code was giving me an error since the type required by the icon prop and the MUI icon itself did not match. Do you know why that would be the case?

Reply:
I believe I already shared the solution to this - use {<LocationOnIcon/>} instead of just passing {LocationOnIcon} and the types should match

Reply:
Yep I was able to fix it! Thank you!